tests/test_checks.py

```diff
@@ -0,0 +1,142 @@
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+from django.core import checks
```

Comment:
sort your imports as per the `isort` output in the tests (see Travis or `tox -e py35-codestyle`)